xds/src/test/java/io/grpc/xds/ControlPlaneRule.java

```diff
@@ -118,14 +126,17 @@ public Server getServer() {
 
   @Override protected void finished(Description description) {
     if (server != null) {
-      server.shutdownNow();
+      System.out.println(">>> shutting down server on port " + server.getPort());
```

Comment:
Is this logging still helpful?

Reply:
There's also the "awaiting termination" and such. I was asking about it as well.

Reply:
Oh yes - removed those now too.